src/embed/IVbrickApi.ts

```diff
@@ -61,12 +61,12 @@ export interface IVbrickWebcastEmbed extends IVbrickBaseEmbed {
 	/**
 	 * Fired when the webcast starts broadcasting.
 	 */
-	on(event: 'broadcastStarted', listener: () => void): void;
+	on(event: 'broadcastStarted', listener: (event: { status: WebcastStatus.Broadcasting }) => void): void;
 
 	/**
 	 * Fired when the webcast stops broadcasting.
 	 */
-	on(event: 'broadcastStopped', listener: () => void): void;
+	on(event: 'broadcastStopped', listener: (event: { status: WebcastStatus.Completed }) => void): void;
```

Comment:
If you add status to these two, also add it to webcastStarted and webcastEnded

Reply:
done